### DISCUSSION TITLE
Is there an easy way to spawn private custom models in hubs?

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#2888] [General Questions & Discussion] [answered]
Is it possible to spawn custom local or locally hosted assets into a custom hubs cloud instance? I know that sketchfab and google poly are out of the box supported. But just wondering how those worked inside the custom hubs code so that i could have a try at spawning custom assets if possible

**Comment by netpro2k** [comment, marked as answer]
You can directly drag and drop `.glb` models into Hubs or paste (ctrl+v) the url to a gltf/glb model publicly accessible on the web.

If you are asking about adding another source to the media search screen (like Sketchfab and Google Poly), that would either require some client changes to hit a custom search service you create, or require some custom backend code to add a new search service to reticulum.

Customizing reticulum is not really supported right now. (It is doable but requires you manually tinkering with habitat packages to stand up your own version of the reticulum service). So if you wanted to do this I would recommend the clientside-only approach.
